CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # Changelog
 
+## [2.0.0]
+
+Full TypeScript support
+
+
+### Component prop changes
+
+*Avatar*
+- add `testID` prop
+
+*Channel*
+- add `additionalKeyboardAvoidingViewProps` prop to allow custom keyboard props
+
+*ChannelListMessenger*
+- remove `setActiveChannel` prop
+
+*ChannelPreviewMessenger*
+- renamed the `latestMessage` prop to `latestMessagePreview`. This name change is more semantic to what the prop does and reduces confusion with the `lastMessage` prop
+
+*MessageContent*
+- removed `retrySendMessage` prop in favor of `retrySendMessage` within `MessagesContext`
+
+*MessageInput*
+- remove `parent` prop to `parent_id` as it needs to be just an id string instead of the entire parent object
+- add `setInputRef` prop to actually allow forwarding of the TextInput ref controls
+
+*MessageSystem*
+- add `formatDate` prop to allow custom date formatting
+
```

Comment:
We also need to add context related changes in ChangeLog

Reply:
@vishalnarkhede yeah Dan already added those back in